### DISCUSSION TITLE
Release of bugfix version 0.7.4

### DISCUSSION THREAD

**Discussion by rafaelmag110** [eclipse-tractusx/tractusx-edc#1496] [General]
Hello,

In light of the current effort to release a bugfix 0.7.4 version on tractusx-edc, let us keep this discussion updated to have transparency on the release progress.

Here is the list of fixes to apply to 0.7.3:

- [x] Backport of [this](https://github.com/eclipse-tractusx/tractusx-edc/pull/1475) PR: Add persistence for data-plane-selector @bmg13 will take care
- [x] Backport of [this](https://github.com/eclipse-tractusx/tractusx-edc/pull/1396) PR: optimize state machines **@zub4t will take care**
- [x] Backport of [this](https://github.com/eclipse-tractusx/tractusx-edc/pull/1456) PR: Adjusted BPN validations on different operators **@zub4t will take care**
- [x] Republishing of SwaggerHub specs including upstream EDC ones.
- [x] Bump EDC dependencies to 0.7.2 which should contain the following fixes:
  - [x] Hotfix: S3 Dataplane does not register any generator function for AmazonS3 type
  - [x] Hotfix: Azure Dataplane does not register any generator function for AzureStorage type
  - [x] Backport of the solution to [this](https://github.com/eclipse-edc/Technology-Aws/issues/384) issue. Still requires a PR though: AWS S3 folder copy not working
  - [x] Backport of [this](https://github.com/eclipse-edc/Technology-Aws/pull/379) PR: Modifies the AWS vault implementation to update existing secrets **@zub4t will take care**
  - [x] Backport of [this](https://github.com/eclipse-edc/Connector/pull/4316) PR: adds order by timestamp **@zub4t will take care**

**Comment by bmg13** [comment]
👍

**Comment by bmg13** [comment]
Discussed offline and BPNL solution can be backported to the 0.7.4 version.

**Reply by rafaelmag110** [reply]
Thanks. I'll add it to the list.

**Comment by rafaelmag110** [comment]
Release was done: https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.7.4
